### PR TITLE
Remove Google Analytics, add Piwik.

### PR DESCRIPTION
**Pull request by thefinn93**
I can give accounts to whoever needs them.

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -15,5 +15,28 @@
...
+
+    We use this to try to understand how far the word about SeaGL has spread.
+
+    We also understand that some people don't like to be tracked, and would highly
+    recommend something like adblock with the privacy filter list.
+    ================================================== -->
+    <!-- Piwik -->
+    <script type="text/javascript">
+      var _paq = _paq || [];
+      _paq.push(["setDomains", ["*.seagl.org"]]);
```
Comment (shanet) [inline _includes/footer.html]: Will this work for just `seagl.org` too?

Comment (thefinn93) [inline _includes/footer.html]: yeah

**Comment by meonkeys** [comment]
Thanks for the PR! I'd like access to view analytics.

@kormoc will you CNAME metrics.seagl.org to analytics.janky.solutions ?

**Comment by kormoc** [comment]
Added. Should be available in an hour or two

**Comment by thefinn93** [comment]
Cool, thanks. Server setup is done, i'll make another PR to update the domain

**Pull request merged**